### DISCUSSION TITLE
Question about Wait duration and syntax.

### DISCUSSION THREAD

**Discussion by WildSnail913** [nirgoren/Eddienput#40] [Technical support]
1. What are the functional differences between the commands in these columns? Which of them are correct/supported syntax?

```
Column A || Column B || Column C || Column D || Column E
2        || 2 W1     || 236+P    || 2 3 6+P  || 2 W1 3 W1 6+P W1
3        || 3 W1     ||          ||          ||
6+P      || 6+P W1   ||          ||          ||
```

I ask because I got different results using some of them. Some even gave inconsistent results for replaying the same script. (I used quarter circles here for brevity, I was testing full circle motions.)

2. I assume some of these inputs are being held less than 1 frame, if so how long are they being held?

**Comment by nirgoren** [comment]
> 1. What are the functional differences between the commands in these columns? Which of them are correct/supported syntax?
> 
> ```
> Column A || Column B || Column C || Column D || Column E
> 2        || 2 W1     || 236+P    || 2 3 6+P  || 2 W1 3 W1 6+P W1
> 3        || 3 W1     ||          ||          ||
> 6+P      || 6+P W1   ||          ||          ||
> ```
> 
> I ask because I got different results using some of them. Some even gave inconsistent results for replaying the same script. (I used quarter circles here for brevity, I was testing full circle motions.)
> 
> 2. I assume some of these inputs are being held less than 1 frame, if so how long are they being held?

`A` should be equivalent to `D`.
`B` should be equivalent to `E`.

C is not supported syntax.

The inputs in the case of `A` and `D` are held for one frame. However, due to Eddienput not being actually synced with the game's clock, depending on the game and its input system this could sometimes result in some dropped inputs.
The inputs in the case of `B` and `E` are held for two frames.

Hope this makes it clear!
